web/source/osk/visualKeyboard.ts

```diff
@@ -679,6 +679,87 @@ namespace com.keyman.osk {
       }
     }
 
+    get width(): number {
+      if(this._width) {
+        return this._width;
+      } else {
+        return this.loadSizeFromCookie().width;
+      }
+    }
+
+    get height(): number {
+      if(this._height) {
+        return this._height;
+      } else {
+        return this.loadSizeFromCookie().height;
+      }
+    }
+
+    protected loadSizeFromCookie(): {width: number, height: number} {
+      let keyman = com.keyman.singleton;
+      let util = keyman.util;
+
+      var c = util.loadCookie('KeymanWeb_OnScreenKeyboard');
```

Comment:
`util.loadCookie` returns an empty object rather than `null` or `undefined` if the cookie is not present. Looking through other places where we use `loadCookie` this seems like a real bug in `loadCookie` and may be causing other problems as well. That means that ll 705-706 are never executed!

Reply:
The other references I can find suggest that there was a spec change at some point, but that there are no real bugs to speak of from this.  We can safely remove lines 705-706.